toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -314,24 +298,22 @@ func (r *RpmRepoCloner) SearchAndClone(cloneDeps bool, singlePackageToClone *pkg
 				continue
 			}
 			// Local sources are listed last, keep searching for the last possible match
-			pkgName = matches[1]
-			logger.Log.Debugf("'%s' is available from package '%s'", singlePackageToClone.Name, pkgName)
+			packageName = matches[1]
```

Comment:
I have noticed this assumption does not always hold, might need to be smarter here.

Reply:
Any suggestions? If its not straightforward I'd prefer to do it in a separate PR.

Reply:
No, not straight forward. Separate PR is fine.